pcap-new.c

```diff
@@ -982,7 +982,7 @@ pcap_t *pcap_open(const char *source, int snaplen, int flags, int read_timeout,
 		break;
 
 	default:
-		strcpy(errbuf, "Source type not supported");
+		pcap_snprintf(errbuf, PCAP_ERRBUF_SIZE, "Source type not supported");
 		return NULL;
```

Comment:
This should probably just be

    strlcpy(errbuf, PCAP_ERRBUF_SIZE, "Source type not supported");

because it's just a copy - no string formatting is being done.